contracts/permissionless/OrderbookReserve.sol

```diff
@@ -751,7 +798,7 @@ contract OrderbookReserve is OrderIdManager, Utils2, KyberReserveInterface, Orde
         int weiDiff = isEthToToken ? (int(newSrcAmount) - int(prevSrcAmount)) :
             (int(newDstAmount) - int(prevDstAmount));
 
-        require(weiAmount >= minNewOrderSizeWei);
+        require(weiAmount >= limits.minNewOrderSizeWei);
```

Comment:
below in require(bindOrderFunds(maker, isEthToToken, int(int(newSrcAmount) - int(prevSrcAmount)))) :
the external casting to int is redundant.